app.rb

```diff
@@ -14,10 +16,15 @@ class App
   # rentals handle
   include RentalHandle
 
+  include ReadData
+
   def initialize
     @books = []
     @people = []
     @rentals = []
+    read_books
+    read_people
+    read_rentals
   end
 
```

Comment:
- [ ] It would be better if the `press any key` functionality works, currently when we press any key we keep seeing the same message. Currently, the `press any key` works as press enter. Let us fix it